code/models/projection_helper_functions.R

```diff
@@ -217,10 +222,32 @@ compute_predicted_cum_cases <- function(full_data, model, policy_variables_used,
       all()
   })
   
-  if(return_no_policy_projection_output){
-    out <- 
-      no_policy_counterfactual_data_storage %>% 
-      group_by(tmp_id) %>% 
+  true_predict <- predict.felm(model, newdata = mmat_actual_for_prediction)
+  true_predict_df <- true_data_subset_for_prediction %>% 
+    mutate(prediction_logdiff = true_predict$fit) %>% 
+    select(tmp_id, date, prediction_logdiff)
+  
+  true_data_storage <- true_data_storage %>% 
+    ungroup() %>% 
+    left_join(true_predict_df, by = c("tmp_id", "date"))
+    
+  if(return_raw_projection_output){
+    datetimes <- no_policy_counterfactual_data_storage %>%
+      group_by_at(.vars = vars(matches("^adm[0-2]_name$"))) %>%
+      summarise(timestep = list(
+        {
+          c(time_steps_per_day, rep(1:time_steps_per_day, times = length(date) - 1))
+        }
+      ),
+      date = list(
+        {
+          c(date[1], rep(date[-1], each = 6))
```

Comment:
@kendonB should this be `each = time_steps_per_day` rather than `each = 6` ?

Reply:
@andyhultgren superstar reviewing! great catch